std/experimental/allocator/gc_allocator.d

```diff
@@ -170,10 +170,13 @@ pure @system unittest
 
         assert(GC.sizeOf(buffer.ptr) == s);
 
-        auto buffer2 = GCAllocator.instance.allocate(s - (s / 2) + 1);
-        scope(exit) () nothrow @nogc { GCAllocator.instance.deallocate(buffer2); }();
+        version (none) // relax condition for the actual GC
```

Comment:
Should specify what the condition is.

Reply:
I updated the comment with more details.